src/tasks/tasks.py

```diff
@@ -415,6 +415,25 @@ def load_data(self, path, max_seq_len):
         log.info("\tFinished loading SST data.")
 
 
+@register_task('hd_cola_npi_sup', rel_path='CoLA/hd_npi_sup')
+@register_task('hd_cola_npi_quessmp', rel_path='CoLA/hd_npi_quessmp')
+@register_task('hd_cola_npi_ques', rel_path='CoLA/hd_npi_ques')
+@register_task('hd_cola_npi_qnt', rel_path='CoLA/hd_npi_qnt')
+@register_task('hd_cola_npi_only', rel_path='CoLA/hd_npi_only')
+@register_task('hd_cola_npi_negsent', rel_path='CoLA/hd_npi_negsent')
+@register_task('hd_cola_npi_negdet', rel_path='CoLA/hd_npi_negdet')
+@register_task('hd_cola_npi_cond', rel_path='CoLA/hd_npi_cond')
+@register_task('hd_cola_npi_adv', rel_path='CoLA/hd_npi_adv')
+@register_task('all_cola_npi', rel_path='CoLA/all_npi')
+@register_task('cola_npi_sup', rel_path='CoLA/npi_sup')
+@register_task('cola_npi_quessmp', rel_path='CoLA/npi_quessmp')
+@register_task('cola_npi_ques', rel_path='CoLA/npi_ques')
+@register_task('cola_npi_qnt', rel_path='CoLA/npi_qnt')
+@register_task('cola_npi_only', rel_path='CoLA/npi_only')
+@register_task('cola_npi_negsent', rel_path='CoLA/npi_negsent')
+@register_task('cola_npi_negdet', rel_path='CoLA/npi_negdet')
+@register_task('cola_npi_cond', rel_path='CoLA/npi_cond')
+@register_task('cola_npi_adv', rel_path='CoLA/npi_adv')
```

Comment:
I suggest moving these to a different task that loads test_full.tsv for the test set. This would make test-set evaluation so much easier.

e.g.,
te_data = load_tsv(self._tokenizer_name, os.path.join(path, 'test_full.tsv'), max_seq_len,
                           s1_idx=3, s2_idx=None, label_idx=1)